example/30_grouped_convnd_fwd_bias_relu_add/CMakeLists.txt

```diff
@@ -8,4 +8,9 @@ add_example_executable(example_grouped_convnd_fwd_bias_relu_add_xdl_bf16 grouped
 target_link_libraries(example_grouped_convnd_fwd_bias_relu_add_xdl_bf16 PRIVATE utility)
 
 add_example_executable(example_grouped_convnd_fwd_bias_relu_add_xdl_int8 grouped_convnd_fwd_bias_relu_add_xdl_int8.cpp)
-target_link_libraries(example_grouped_convnd_fwd_bias_relu_add_xdl_int8 PRIVATE utility)
\ No newline at end of file
+target_link_libraries(example_grouped_convnd_fwd_bias_relu_add_xdl_int8 PRIVATE utility)
+
+if(USE_BITINT_EXTENSION_INT4)
+  add_example_executable(example_grouped_convnd_fwd_bias_relu_add_xdl_int4 grouped_convnd_fwd_bias_relu_add_xdl_int4.cpp)
+  target_link_libraries(example_grouped_convnd_fwd_bias_relu_add_xdl_int4 PRIVATE utility)
```

Comment:
You don't have to link `utility` library manually. It's already done in `add_example_executable`. Please do fix all the occurences in this CMakeLists file.

Reply:
Fixed in [439db12](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/375/commits/439db121bb7109de4d70ac6173824e0ce83823fa)